components/dropdown/dropdown-content-mixin.js

```diff
@@ -205,6 +216,9 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		this.noPaddingFooter = false;
 		this.noPaddingHeader = false;
 		this.noPointer = false;
+		this.mobileBreakpointOverride = 616;
+		this.mediaQueryList = window.matchMedia(`(max-width: ${this.mobileBreakpointOverride - 1}px)`);
```

Comment:
Does this need to be set here, or since it's getting set again in `connectedCallback` is that enough?